certified-connectors/WenDocs Linker/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API_Key",
+          "description": "provide the API_Key obtained from WenDocs Corporation[www.wendocs.com]",
+          "tooltip": "the API_Key for access publish api service",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API_Secret",
+          "description": "the API_Secret for access publish api service",
+          "tooltip": "provide the API_Secret obtained from WenDocs Corporation[www.wendocs.com]",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#92D050",
+    "capabilities": ["actions"],
+    "publisher": "WenDocs Ltd",
+    "stackOwner": "Beijing Huawen Yundao Technology Co., Ltd"

```

Comment:
@JulioWendocs Please provide us any document which shows relationship b/w the stackOwner and publisher ?